django/cantusdb_project/main_app/tests/make_fakes.py

```diff
@@ -310,22 +303,20 @@ def make_fake_segment(name: str = None, id: int = None) -> Segment:
     return segment
 
 
-def make_fake_sequence(
-    source=None, title=None, incipit=None, cantus_id=None
-) -> Sequence:
+def make_fake_sequence(source=None, title=None, cantus_id=None) -> Sequence:
     """Generates a fake Sequence object."""
     if source is None:
         source = make_fake_source(segment_name="Bower Sequence Database")
     if title is None:
         title = faker.sentence()
-    if incipit is None:
-        incipit = title[:INCIPIT_LENGTH]
     if cantus_id is None:
         cantus_id = make_random_string(6, "0123456789")
     sequence = Sequence.objects.create(
         title=title,
+        # most sequences in the database have no standard-spelling fulltext,
```

Comment:
They do? I don't think this materially impacts anything, but I find none that have it...

Reply:
You're right. I made an assumption; thanks for actually verifying this.